deployment_script.py

```diff
@@ -44,6 +44,7 @@ def main():
         add_eval_3_adm_data(mongoDB)
         add_admin_user_role(mongoDB)
         update_scenario_index(mongoDB)
```

Comment:
We should remove this other calls because those scripts have run, we should also probably remove the probe matcher script from running now that it has run successfully.